packages/manager/src/features/Linodes/LinodeCreatev2/Tabs/Marketplace/utilities.ts

```diff
@@ -132,9 +132,50 @@ const getDoesStackScriptMatchQuery = (
  * @param stackscript The StackScript to compare aginst
```

Comment:
```suggestion
 * @param app The Marketplace app to compare against
```
Nit: if we're renaming the param we should update this too.